### ISSUE TITLE
Cannot Select, Copy, Paste, Use Keyboard Shortcuts, and etc. 

### ISSUE THREAD

**Issue by cryptoguard** [particl/particl-desktop#1251]
**OS:** MacOS 10.14
**Client:** 
<img width="247" alt="screen shot 2018-11-12 at 7 46 48 pm" src="https://user-images.githubusercontent.com/27031337/48383440-be3cf300-e6b3-11e8-8fd3-1a77c69e4d09.png">

This appears to be a Particl Marketplace-wide issue. In this screenshot: 

<img width="982" alt="screen shot 2018-11-12 at 7 38 01 pm" src="https://user-images.githubusercontent.com/27031337/48383136-88e3d580-e6b2-11e8-818a-38bf973410e0.png">

Here, it is impossible to highlight the text (i.e. to copy-paste in a translator), or copy-paste it. It is also impossible to use keyboard shortcuts such as cmd + a, cmd + c, cmd + v on MacOS. 

When creating a listing: 

<img width="561" alt="screen shot 2018-11-12 at 7 45 23 pm" src="https://user-images.githubusercontent.com/27031337/48383398-8a61cd80-e6b3-11e8-813d-6a818161ba11.png">

It is possible to paste text from an outside source using cmd + v. However, it is not possible to select all using cmd + a, copy text originating from Particl Desktop by highlighting with cursor and doing cmd + c or right-click. It is also impossible to do cmd + z. It is possible to highlight with cursor and delete doing when pressing _delete_  on Mac keyboard. 

## tl;dr

All in all, would be much more useful if it was possible to always use keyboard shortcuts when in a text box and always be able to highlight with the cursor and cmd + c/cmd + v.

**Comment by zaSmilingIdiot** [comment]
As mentioned in #1439 :

Mac builds don't have the default keyboard shortcuts enabled when building via electron. There are a couple of "workarounds" that we are looking into, so should have this fixed up shortly.

**Comment by zaSmilingIdiot** [comment]
A change for this has been provided. New testnet (and the next mainnet) release builds should all include the change.
